src/lib/components/data-graphic/guides/Axis.svelte

```diff
@@ -173,12 +174,18 @@ This component will draw an axis on the specified side.
     // use graphicWidth or graphicHeight
     // do we ensure different spacing in one case vs. another?
     tickCount = ~~(axisLength / 20);
-    tickCount = Math.max(2, ~~(tickCount / 100));
+    const maxTickCountThreshold = 2 + ~~($plotConfig.width / 250);
+    tickCount = Math.max(maxTickCountThreshold, ~~(tickCount / 100));
+
+    ticks = scale.ticks(tickCount);
+    if (ticks.length === 1 && $plotConfig.width >= 250) {
```

Comment:
when this gets < 250, I see this happens:
1. the domains disappear. Seems like we could just have the domains in this case, right?
2. The y axis also seems to reduce the number of ticks. Wonder what's causing that!

![rill-developer-axis-disappears](https://user-images.githubusercontent.com/95735/180796970-d7800214-b051-4efe-be3a-088cc5bec7d3.gif)